app/pages/events/$id.mjs

```diff
@@ -1,9 +1,35 @@
+import { marked } from "marked"
+
 export default function ({ html, state = {} }) {
   let { store = {} } = state
-  let { event } = store
+  let event = store.events[0]
+  let hasTalks = event.talks && event.talks.length > 0
+
+  if (event) {
   return html`
-    <page-layout title=${event.title}>
-      <pre>${JSON.stringify(event, null, 2)}</pre>
+  <page-layout title=${event.title}>
+      <h3>${event.title}</h3>
+      ${event.description && `<p>${marked(event.description)}</p>` }
+      <h4>Thanks to our Sponsors ❤️</h4>
+      ${event.sponsors ? html`<list-sponsors></list-sponsors>` : null}
+      ${hasTalks ? html`<list-talks event_id="${event.id}"></list-talks>`: html`<p>No talks were given durring this event.</p>`}
+      <div class="">
+        <a target="_self" href="/events">
+          See all past events <i class="fa-solid fa-arrow-right"></i>
+        </a>
+      </div>
     </page-layout>
   `
+  } else {
+    return html`
```

Comment:
We should return a 404 HTTP status code here so browsers know this isn't a valid URL. We return 404 codes in the API for this route so api/events/$id.mjs.

See api/$$.mjs for an example of how to return a 404. Once we do that we should be able to remove the `if (event)` check here

Reply:
✅